server/controllers/appointment.js

```diff
@@ -0,0 +1,39 @@
+const appointmentsRouter = require("express").Router();
+const { Appointment } = require("../models/");
+
+// Create Appointments
+// How to make this an authenticated route?
+appointmentsRouter.post("/", (request, response, next) => {
+    const body = request.body;
+    const time = new Date(body.time);
+    console.log(typeof time)
```

Comment:
remove console.logs

Reply:
absolutely, I had them in there for debugging originally.